CMakeLists.txt

```diff
@@ -81,17 +80,27 @@ if(NB_CREATE_INSTALL_RULES AND NOT CMAKE_SKIP_INSTALL_RULES)
 
   if(NB_USE_SUBMODULE_DEPS)
     install(
-      DIRECTORY ext/robin_map/include/
-      DESTINATION "${NB_INSTALL_EXT_DIR}/robin_map/include"
-    )
-    install(
-      FILES ext/robin_map/CMakeLists.txt
-      DESTINATION "${NB_INSTALL_EXT_DIR}/robin_map"
+      FILES ext/robin_map/include/tsl/robin_map.h
+            ext/robin_map/include/tsl/robin_hash.h
+            ext/robin_map/include/tsl/robin_growth_policy.h
+      DESTINATION "${NB_INSTALL_EXT_DIR}/robin_map/include/tsl"
```

Comment:
This matches the behavior of the previous `setup.py`. It excludes `robin_set.h`. Is this intentional? Or should it install `*.h` from `robin_map`?